### DISCUSSION TITLE
New pattern for an eyepatch

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1621] [Forum]
**What is it that you would like to see happen?**

Whether you need it as a utility or costume having a pattern for an eye patch would be helpful and a good first pattern if you wanted an excuse to try making a pattern.

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**

Ideally would be great to make it work for both eyes rather than only working for the left or right eye but could potentially be an option to let the user choose. Could use the strap method which was used for [Florence](https://freesewing.org/docs/patterns/florence/) as this is helpful if you don't have access to any stretchy material.

This will likely be one of our first patterns to see what the contribution process is like & how to generate parametric patterns. Anybody who is new to contributing or interested in helping is welcome to join. See this video if you need some guidance on how you may be able to help - https://www.youtube.com/watch?v=UWA4wyacY2A

More details about eye patches can be found here - https://en.wikipedia.org/wiki/Eyepatch

Suggested name Eve the eye patch

**Comment by darigovresearch** [comment]
@bobgeorgethe3rd you mentioned on the contributor call on the 27th various types of variations/features that eye patches can have, can you outline them below in a comment to help guide this pattern design?

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions

**Comment by darigovresearch** [comment]
Reopening discussion from the community call https://github.com/freesewing/freesewing/discussions/5532

**Reply by woutervdub** [reply]
OMG, I should have seen this when I made `gozer`!

Proposed name: `Elle` (Driver) ;-)

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
